howto/container/logs.md

```diff
@@ -1,3 +1,20 @@
+## Runtime logs
```

Comment:
```suggestion
You can view the Anbox and the Android system logs while a container is running, or check the collected logs if a container fails.
[note type="information" status="Note"]AMS does not support runtime log collection. Logs are currently only being collected from a container that failed to start or had an error at runtime.[/note]

## View runtime logs
```

Reply:
This moves the note from the bottom up. Do I understand it correctly that the note is still true with the addition in this PR, but that runtime logs can only be **viewed** and are not **collected**?

Reply:
We can view runtime logs (while the container is running) and collected logs (when the container is stopped). 

Makes sense to drop the note.